src/components/NodeToolbar/AccessibilityEditor/saveStatus.js

```diff
@@ -71,6 +72,7 @@ function save<T>(options: SaveOptions<T>): Promise<Response> {
       throw response;
     })
     .then(json => {
+      debugger;
```

Comment:
@opyh This debugger statement is now in master right?

Reply:
Oops. Will remove it.